docs/src/guides/run-genbank-ingest-on-terra.rst

```diff
@@ -0,0 +1,65 @@
+*********************************************
+Ingest SARS-CoV-2 data from GenBank on Terra
+*********************************************
+
+We have provided two pipelines for importing data into Terra:
+
+* **GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.**
+* GISAID Ingest - pull a private dataset if a user has their own API endpoint, username, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagrammed below:
+
+.. image:: ../images/terra-ingest.png
+
+This guide describes the **GenBank Ingest**.
+
+Import the GenBank ingest wdl workflow from Dockstore
+======================================================
```

Comment:
Suggestion: Add a note that this is a one-time "setup" per user. I imagine users will want to run the workflow more than once, but it's not necessary to follow these steps more than once.

Reply:
One-time setup per workspace, but I understand what you mean. Done with c4dd56830f6ed248f97f1d4a0f13eeb08394dd1d